### PR TITLE
[READY] Setup Documentation Update

### PR DESCRIPTION
**Pull request by irvingywang**
- Requires testing.
- Open to revisions, this is a draft.
- Does not include linux setup instructions.

### CODE REVIEW

**Review by mcgalliard — DISMISSED** [review]

**Comment by irvingywang** [comment]
MacOS install tested and working

**Review by ColexDev — DISMISSED** [review]

File: setup.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+10. Install [CMake](https://cmake.org/install/): Build system generator. This takes care of making all of the build files needed to compile the project.
+
+    - Windows: `pacman -S mingw-w64-x86_64-cmake`.
+    - MacOS: `brew install cmake`.
+    <!-- On some MacOS versions, CMake will install as a GUI only, follow the `Tools > Install Command Line Tools` tip inside CMake to fix this. -->
+
+11. Install [Ninja](https://ninja-build.org/): Small & fast build system used by CMake.
+
+    - Windows: `mingw-w64-x86_64-ninja`.
```
Comment (zyonse) [inline setup.md:56]: Some commands are missing "pacman -S"

**Review by zyonse — CHANGES_REQUESTED** [review]

File: setup.md (hunk 1 above)
Comment (irvingywang) [reply setup.md]: fixed, good catch

**Review by zyonse — APPROVED** [review]
Should be ready now

**Comment by irvingywang** [comment]
@mcgalliard 1 approval needed to merge

**Review by ColexDev — APPROVED** [review]

**Review by AdityaAsGithub — APPROVED** [review]

**Pull request merged**